cicd/internal/flags/common-flags_test.go

```diff
@@ -52,3 +53,23 @@ func TestModulesToBuild(t *testing.T) {
 		}
 	}
 }
+
+func TestDefaultExcludedSubModules(t *testing.T) {
```

Comment:
the refactor and tests makes it convenient for future workflow split (just need to add new moduleMap entry)